### PR TITLE
 Added argument checks for exported input functions.

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: libfreerdp/core/input.c [hunk 1]
```diff
@@ -479,6 +539,9 @@ BOOL input_recv(rdpInput* input, wStream* s)
 {
 	UINT16 i, numberEvents;
 
+	if (!input || !s)
```
Comment (mfleisz) [inline libfreerdp/core/input.c]: I think that stream NULL pointer check is not necessary as this function is called from the transport layer where stream creation is already checked and ensured.

Comment (akallabeth) [inline libfreerdp/core/input.c]: does not hurt either and the symbol is exported (at least libfreerdp internally)

**Comment by bmiklautz** [comment]
+1 if the freerdp-bot is also ok with it ;)

**Comment by hardening** [comment]
I think it's ready to merge when @freerdp-bot will have told us the result of the build

**Comment by akallabeth** [comment]
@freerdp-bot Be a nice bot and test this again

**Comment by bmiklautz** [comment]
@freerdp-bot just test!

**Pull request merged**